src/main/java/org/lsc/plugins/connectors/james/JamesDao.java

```diff
@@ -187,5 +194,82 @@ public boolean deleteAlias(User user) {
 				.filter(alias -> !sourceAliases.contains(alias))
 				.collect(Collectors.toList());
 	}
+
+	public boolean addUser(User user) {
+		Response response = usersClient
+			.path(user.email)
+			.request()
+			.header(HttpHeaders.AUTHORIZATION, authorizationBearer)
+			.put(Entity.text("{\"password\":\"secret\"}"));
```

Comment:
Generate a random password of at least 24 chars please. common-text have such utilities.

Reply:
I remember the requirement in the ticket. 
But I think, why we need a random password while we never use them after? Just a static password for all is better?

Reply:
> But I think, why we need a random password while we never use them after?

 - What if an admin miss configure the Cassandra Users repository instead of the LDAP one, and all passwords become `secret`?
 
  - We could be using that LSC plugin with the CassandraUsersRepository. JMAP users could authenticate via JWT but we would require to reset a password to authenticate via IMAP/SMTP. Compared to the actual pattern that we are using (provisioning on the first JMAP request, where we create the user with a random password), LSC would be an improvement as it allows to remove users.